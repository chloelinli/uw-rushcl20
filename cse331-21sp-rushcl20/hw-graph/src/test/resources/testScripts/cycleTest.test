CreateGraph graphA
AddNode graphA n1
AddNode graphA n2
AddNode graphA n3
AddEdge graphA n1 n2 e1
AddEdge graphA n2 n3 e2
AddEdge graphA n3 n1 e3
ListChildren graphA n1
ListChildren graphA n2
ListChildren graphA n3