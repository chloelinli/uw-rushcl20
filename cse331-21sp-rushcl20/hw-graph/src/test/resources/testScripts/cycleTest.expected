created graph graphA
added node n1 to graphA
added node n2 to graphA
added node n3 to graphA
added edge e1 from n1 to n2 in graphA
added edge e2 from n2 to n3 in graphA
added edge e3 from n3 to n1 in graphA
the children of n1 in graphA are: n2(e1)
the children of n2 in graphA are: n3(e2)
the children of n3 in graphA are: n1(e3)